### PR TITLE
Doc Features: Repository

### PR DESCRIPTION
**Pull request by pushker001**
Helps user to understand the features of Helm-dashboard
https://github.com/komodorio/helm-dashboard/issues/255

- [X] The title of my pull request is a short description of the changes
- [X] This PR relates to some issue:

### CODE REVIEW

**Review by undera — CHANGES_REQUESTED** [review]
From the structure from #255, you have described "add new repo" piece, which is fine. However, it is better to describe the purpose of whole "Repository" section first, with screenshot of where you find it in the main menu. It is also useful linking the Helm's doc on what are repositories (this one https://helm.sh/docs/topics/chart_repository/).
Then, when reader has discovered the top-level concept, you proceed with describing how to add repository, how to update and remove it. How to filter the available charts list and how to proceed with installing one (but not describe the install process which will have an own section).

As for "add repository" description, it is a bit slim. It would be better to explain in words that user needs to provide repository name and URL, and optional username and password. Just a tiny bit more elaborate and elegant description.

Let's improve it!

**Comment by pushker001** [comment]
@undera The changes I have done is not correct??

File: FEATURES.md [hunk 1]
```diff
@@ -6,3 +6,20 @@ To close Helm-dashboard, click on the button in the rightmost corner of the scre
...
+Essentially, a repository is a location where charts are gathered and can be shared. If you want to learn more about repositories, [click here](https://helm.sh/docs/topics/chart_repository/). You can find the repository in the home section, as depicted in the figure.
+![Repository3](images/screenshot_repository3.png)
+
+You can add the repository by clicking on 'Add Repository', as shown in the figure.
+![Repository](images/screenshot_repository.png)
+
+After completing that step, enter the following data: the repository name and its URL. You can also add the username and password, although this is optional.
+![Repository2](images/screenshot_repository2.png)
+
+If you want to update your repository, delete it, and then filter your chart, you can do so as shown in the figure.
```
Comment (undera) [inline FEATURES.md:20]: This part is a bit meaningless. Updating repository is a separate action, not connected to deletion and filtering. Deletion is again a separate action and deserves own couple of lines of doc. Filtering is a whole own story, related to how you browse the list of available charts, that you can filter that list and trigger installing of chart. 

Bottom line: describe individual features of repository actions, not whole block/

Comment (pushker001) [reply FEATURES.md:20]: so I need to write about filter, update and delete separately only ? all other things are okay?

Comment (undera) [reply FEATURES.md:20]: Yes, you have improved the above pieces and those are fine. Good job!

We just need to improve the remaining parts.

File: FEATURES.md [hunk 2]
```diff
@@ -6,3 +6,22 @@ To close Helm-dashboard, click on the button in the rightmost corner of the scre
...
+After completing that step, enter the following data: the repository name and its URL. You can also add the username and password, although this is optional.
+![Repository2](images/screenshot_repository2.png)
+
+Updating means refreshing your repository. You can update your repository as shown in the figure.
+![Repository4](images/screenshot_repository4.png)
+
+If you want to remove your repository from the Helm dashboard, click on the 'Remove' button as shown in the figure.
+![Repository5](images/screenshot_repository5.png)
+
+If you want to install a particular chart from the list of available charts, use the filter as shown in the figure.
```
Comment (undera) [inline FEATURES.md:26]: Now, let's improve this section, it is a bit misleading now. To install the chart, you need to hove the line in table, then the "Install" button will appear. Clicking on this button will trigger the install.

The "filter" is just a way to find the desired chart quicker in the table.

File: FEATURES.md [hunk 3]
```diff
@@ -6,3 +6,22 @@ To close Helm-dashboard, click on the button in the rightmost corner of the scre
...
+After completing that step, enter the following data: the repository name and its URL. You can also add the username and password, although this is optional.
+![Repository2](images/screenshot_repository2.png)
+
+Updating means refreshing your repository. You can update your repository as shown in the figure.
+![Repository4](images/screenshot_repository4.png)
+
+If you want to remove your repository from the Helm dashboard, click on the 'Remove' button as shown in the figure.
+![Repository5](images/screenshot_repository5.png)
+
+Use the filter option to find the desired chart quicker from the list of charts.
```
Comment (undera) [inline FEATURES.md:26]: Now it looks fine. The whole section just misses the item about how to trigger chart install (by hovering the desired chart and clicking "install".

Comment (pushker001) [reply FEATURES.md:26]: ok I am on it

**Comment by undera** [comment]
Very nice @pushker001

**Pull request merged**